keps/sig-architecture/2140-receipts/README.md

```diff
@@ -0,0 +1,266 @@
+  - [Graduation Criteria](#graduation-criteria)
+    - [Alpha](#alpha)
+    - [Alpha -&gt; Beta Graduation](#alpha---beta-graduation)
+    - [Future Graduations](#future-graduations)
+- [Implementation History](#implementation-history)
+<!-- /toc -->
+
+## Release Signoff Checklist
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+- [x] (R) Enhancement issue in release milestone, which links to KEP dir in [kubernetes/enhancements] (not the initial KEP PR)
+- [ ] (R) KEP approvers have approved the KEP status as `implementable`
+- [ ] (R) Design details are appropriately documented
+- [-] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [x] (R) Graduation criteria is in place
+- [-] (R) Production readiness review completed
+- [-] Production readiness review approved
+- [-] "Implementation History" section is up-to-date for milestone
+- [-] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation—e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+[kubernetes.io]: https://kubernetes.io/
+[kubernetes/enhancements]: https://git.k8s.io/enhancements
+[kubernetes/kubernetes]: https://git.k8s.io/kubernetes
+[kubernetes/website]: https://git.k8s.io/website
+
+## Summary
+
+
+Introduce a new git based artifact, and supporting tooling, to support automated enhancement collection and tracking processes for Kubernetes Releases.
+
+
+## Motivation
+
+When a Kubernetes release begins, the release team currently begins enhancement collection by:
+
+* Reviewing all open Enhancement issues
+* Reviewing previous issue comments to determine activity
+* [Adding new comments, milestone and manually applying labels to get confirmation of inclusion in the release](https://github.com/kubernetes/enhancements/issues/1979#issuecomment-691394283)
+
+This is very time intensive for the release team and leads to many instances where notifications are missed, ultimately resulting in untracked Exceptions and rushed Exception Requests.
+
+During the implementation phase of the release, more comments and back and forth conversation occurs on the Enhancement issue, including:
+
+* tracking of kubernetes/kuberentes PRs[[1](https://github.com/kubernetes/enhancements/issues/1929#issuecomment-726228115)]
+* tracking kubernetes/website PRs[[2](https://github.com/kubernetes/enhancements/issues/2000#issuecomment-722752540)].
+
+This is also done simply by referencing the KEP from the kubernetes/kubernetes PR.
+
+When a milestone, such as enhancement freeze or code freeze occur, the issue is updated to reflect inclusion or removal from the milestone and we again leverage comments and manually applied labels to convey status, as an example consider [Build-in API Types Misses Code Freeze](https://github.com/kubernetes/enhancements/issues/1929#event-3991979549).
```

Comment:
Thinking about it, indeed it can get confused with the release version milestones like v1.20, v1.21, and so on. I will refactor it to `release cycle stage` for now.